src/buffer_logger/buffer_logger.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABC, abstractmethod
+
+
+class BufferLogger(ABC):
+    def __init__(self, config, config_type):
+        self.config = config
+        self.config_type = config_type
+        self.directory = './logs/'
```

Comment:
We wilde toch een optie hebben om het naar een USB stick oid te schrijven. Als dit bijvoorbeeld op een Pi zou draaien, moet je eerst verbinding maken met de Pi om het bestand te zoeken (je kan niet gewoon vanaf de SD kaart lezen zo ver ik weet).

kan je default `'./logs/'` hebben, maar ook bij detectie van een USB daar (ook?) te schrijven? Ik weet niet of het autodetect van een USB storage device veel werk is of niet in Python

Reply:
Ik maak wel een issue hiervoor